test/base/EIP7702ProxyBase.sol

```diff
@@ -106,15 +106,8 @@ abstract contract EIP7702ProxyBase is Test {
      * @return The target address (for convenience)
      */
     function _deployProxy(address target) internal returns (address) {
-        // Deploy proxy normally first to get the correct immutable values
-        EIP7702Proxy proxy = new EIP7702Proxy(
-            address(_implementation),
-            _initSelector,
-            _nonceTracker
-        );
```

Comment:
same here, noticed there was no need for an additional proxy template to reference when creating certain test situations